src/components/App.js

```diff
@@ -16,25 +19,100 @@ import '../style.css';
 		*Provide form to input repo information dynamically, and display related issues
 */
 
+const options = [
+    {
+      label: "All",
+      value: "all",
+      size: "80px"
+    },
+    {
+      label: "Open",
+      value: "open",
+      size: "100px"
+    },
+    {
+      label: "Closed",
+      value: "closed",
+      size: "110px"
+    },
+  ];
+
 const App = () => {
     
     const [issue, setIssue] = useState([]);
+    const [label, setLabel] = useState([]);
+    const [pageNumber, setpageNumber] = useState(0);
+    const [githubState, setgithubState] = useState(options[0].value); // for filter open/closed/all
+
+    const loadIssues = async (pageNumber) => {
+        let gitState = '';
+            if(githubState === 'all') {
+                gitState = '&state=all'
+            } else {
+                gitState = '&state=' + githubState.value
+            }
 
-        const loadIssues = async () => {
-            const response = await github.get();
+        if (pageNumber === 0) {
+            setpageNumber(1)
+            const response = await github.get('issues?page=' + pageNumber +gitState);
+            setIssue(response.data)
+        } else {
+            const response = await github.get('issues?page=' + pageNumber +gitState);
+            if(response.data.length  === 0){
+                setpageNumber(pageNumber -1)
+                const response = await github.get('issues?page=' + pageNumber +gitState);
+                setIssue(response.data)
+            }
             setIssue(response.data)
         }
+    }
         
-    loadIssues();
-    //add pagination feature in content component
+    const loadLabel = async () => {
+        const response = await github.get('labels');
+        setLabel(response.data)
+    }
+
+    useEffect(() => {
+        loadIssues(pageNumber);
+    }, [pageNumber, githubState]);
+
+    useEffect(() => {
+        loadLabel();
+    },[])
+
         return (
             <div className='ui container'>
                 <Header />
+                <div className="ui grid" id="search-label-content">
+                    <div className="ten wide column" id="search-label-content-column">
+                        <div className="ui left action left icon input fluid">
+                            <DropDown selected={githubState} options={options} onSelectedChange={setgithubState} />
+                            <i className="search icon " id="icon-search" style={{ marginLeft:githubState.size }}></i>
```

Comment:
```suggestion
                              <i className="search icon " id="icon-search" style={{ marginLeft:githubState.size }} />
```
indention and use single tag